crates/bevy_pbr/src/alpha.rs

```diff
@@ -2,16 +2,19 @@ use bevy_ecs::{component::Component, reflect::ReflectComponent};
 use bevy_reflect::std_traits::ReflectDefault;
 use bevy_reflect::Reflect;
 
-// FIXME: This should probably be part of bevy_render2!
-/// Alpha mode
+/// Sets how a PBR material's base color alpha channel is used for transparency.
 #[derive(Component, Debug, Default, Reflect, Copy, Clone, PartialEq)]
 #[reflect(Component, Default)]
 pub enum AlphaMode {
+    /// Base color alpha values are overridden to be fully opaque (1.0).
     #[default]
     Opaque,
     /// An alpha cutoff must be supplied where alpha values >= the cutoff
-    /// will be fully opaque and < will be fully transparent
+    /// will be fully opaque (1.0) and < will be fully transparent (0.0).
```

Comment:
```suggestion
    /// will be fully opaque (1.0) and less than that will be fully transparent (0.0).
```
I think we can spare a few more words here

Reply:
I'd do the same thing for the line above, but github won't let me create a suggestion with it.

Reply:
Wow, using full word actually makes this so much more readable